### ISSUE TITLE
 AssemblyInformationalVersion taken as AssemblyVersion

### ISSUE THREAD

**Issue by haf** [Microsoft/visualfsharp#298]
Example:

```
namespace Loggly.Zipkin.Example
open System.Reflection
open System.Runtime.CompilerServices
open System.Runtime.InteropServices
[<assembly: AssemblyTitle("Intelliplan.Loggly.Zipkin.Example")>]
[<assembly: AssemblyDescription("Assembly is a part of Intelliplan's Shared assemblies")>]
[<assembly: AssemblyConfiguration("RELEASE")>]
[<assembly: AssemblyCompany("Intelliplan International AB")>]
[<assembly: AssemblyProduct("Intelliplan.Loggly.Zipkin.Example")>]
[<assembly: AssemblyCopyright("(c) Intelliplan International 2014")>]
[<assembly: AssemblyVersion("1.2.0.0")>]
[<assembly: AssemblyFileVersion("1.2.0.0")>]
[<assembly: AssemblyInformationalVersion("1.2.0.5f0cfa")>]
()
```

Output from compiler:

```
5>FSC: warning FS2003: An AssemblyVersionAttribute specified version '1.2.0.5f0cfa', but this value is invalid and has been ignored
```

See https://github.com/fsharp/fsharp/issues/289

**Comment by KevinRansom** [comment]
It.looks.like the version validation code needs to be parameterized so that it reports the correct attribute name.

**Comment by enricosada** [comment]
is not only that, see the fsharp/fsharp#289 is not only the warning message wrong, also the fallback

**Comment by latkin** [comment]
This has already been fixed in F# 4.0, in http://visualfsharp.codeplex.com/SourceControl/network/forks/esada/fsharp/contribution/7331 The warning message is now properly parameterized.

@enricosada if there are remaining functional issues, please open a new issue with details on expected and actual behavior.
